bench/script/loop.plutus

```diff
@@ -0,0 +1,5 @@
+{
+    "type": "PlutusScriptV1",
+    "description": "",
+    "cborHex": "588d588b01000033322233223222322533530083322333573466e2000800403002d40092080897a13263530063357389211572656465656d6572206973203c2031303030303030000074984c01540084dd680099000991800800911991299a9803999ab9a3370e002904044bd0048040803099180180099b8100148008c00800800448004800448800848800480041"
```

Comment:
I still wish we could move away from pre-compiled Plutus scripts.

Not in the least because the compiler does improve.

Reply:
This is strictly only used by me for testing the stuff locally.
I often run the test-scripts.json directly and by-pass the tx-generator-service.
The tx-generator-service the latest build result.

Reply:
OIC, thanks for explaining Marc!